warehouse/internal/repo/upload.go

```diff
@@ -1357,3 +1357,68 @@ func (u *Uploads) update(
 	}
 	return nil
 }
+
+func (u *Uploads) GetFirstAbortedUploadsInContinuousAborts(ctx context.Context, workspaceId string) ([]model.FirstAbortedUploadResponse, error) {
+	outputColumns := "id, source_id, destination_id, created_at, first_event_at, last_event_at"
+
+	stmt := fmt.Sprintf(`
+	WITH wh_uploads_with_last_successful_upload AS (
+		SELECT 
+			%s,
+			status,
+        	MAX(CASE WHEN status = 'exported_data' THEN created_at ELSE NULL END) OVER (PARTITION BY destination_id) AS last_successful_upload
+		FROM 
+			`+uploadsTableName+`
+		WHERE workspace_id = $1
+		AND created_at >= NOW() - INTERVAL '30 day'
```

Comment:
Does interval needs to be hard-coded. I thought control-plane would be sending that information.
```suggestion
		AND created_at >= NOW() - $2 * INTERVAL '1 DAY'
```

Reply:
Added a start param